Bio Sheet (#1122)
planetary-social/nos: CHANGELOG.md

## [Unreleased]

> - On the Profile screen, open an Profile dialog to display the full bio. 

Reviewer: maybe this?
```suggestion
- On the Profile screen, open a sheet to display the full bio.
```
